app/styles/app.scss

```diff
@@ -219,12 +219,6 @@ tr.disabled {
   height: 95% !important;
 }
 
-.vl-modal {
```

Comment:
Removing this results in select-options not being shown. Try opening the modals, for example 'Nieuwe agenda aanmaken'.
We're using Ember-Power-Select for the index here. Either lower this index or bump the z-index of the power-select.

Reply:
Very good, seems to work perfectly for the creates.
Could you check the upload of the files for example: http://localhost:4200/agenda/5DA5ADD557D6EC0008000007/documenten

Try uploading a file and select a document-type. This one is still broken for me.

Reply:
Hmm seems to work for me

Reply:
<img width="868" alt="Screenshot 2019-10-17 at 16 17 19" src="https://user-images.githubusercontent.com/20270258/67017336-e8840580-f0f9-11e9-9205-5a03e49e6ea8.png">
You mean this right?

Reply:
<img width="853" alt="Screenshot 2019-10-17 at 16 25 21" src="https://user-images.githubusercontent.com/22617224/67017965-cdfe5c00-f0fa-11e9-9aa8-4ca2ade6278d.png">
If I click it, I don't get any options..

Reply:
And is there a z-index on this element: .ember-basic-dropdown-content--left.ember-basic-dropdown-content--below.ember-basic-dropdown--transitioned-in

Reply:
I don't know how to change it because I can't find it in the code and I can't see the problem, could you give me some more direction?

Reply:
I think I found a fix. I'll just merge the PR and commit the fix with it.
Weird that we can't reproduce it on both machines though.

Reply:
Okay, thanks!